cmd/podman/exec.go

```diff
@@ -47,6 +49,7 @@ func init() {
 	flags.BoolVarP(&execCommand.Tty, "tty", "t", false, "Allocate a pseudo-TTY. The default is false")
 	flags.StringVarP(&execCommand.User, "user", "u", "", "Sets the username or UID used and optionally the groupname or GID for the specified command")
 
+	flags.IntVar(&execCommand.PreserveFDs, "preserve-fds", 0, "Pass N additional file descriptors to the container")
```

Comment:
I think flatpak passes in the specific numbers of the FDs in question, and allows the flag to be specified multiple times. Not sure if compatibility with them is important here.

Reply:
I think we should rather look at what runc already does, it supports `--preserve-fds`, but only for create/run at the moment.

Reply:
If this is runc's behavior, matching it SGTM